pytorch_lightning/trainer/progress.py

```diff
@@ -28,10 +30,10 @@ class Tracker:
 
     Attributes set to ``None`` are treated as unused and are restricted.
     """
-    ready: Optional[int] = 0
-    started: Optional[int] = 0
-    processed: Optional[int] = 0
-    completed: Optional[int] = 0
+    ready: Optional[int] = field(default_factory=int)
+    started: Optional[int] = field(default_factory=int)
+    processed: Optional[int] = field(default_factory=int)
+    completed: Optional[int] = field(default_factory=int)
```

Comment:
This is not necessary. The reference issues you were experiencing were because they were classes. This doesn't happen to primitives (ints in this case)

Reply:
Thanks !